eas.json

```diff
@@ -0,0 +1,30 @@
+{
+  "cli": {
+    "version": ">= 2.0.0"
+  },
+  "build": {
+    "development": {
+      "developmentClient": true,
+      "distribution": "internal"
+    },
+    "preview": {
+      "distribution": "internal",
+      "channel": "preview"
+    },
+    "production": {
+      "channel": "production"
+    }
+  },
+  "submit": {
+    "preview": {
+      "ios": {
+        "appleId": "app_store@nearform.com",
+        "ascAppId": "1643969791",
+        "appleTeamId": "VPLG3LLGBN"
```

Comment:
let's make extra sure this is not sensitive information and especially that we _need_ it to be here

Reply:
it's not needed with our setup. removed ✅